packages/openneuro-components/src/dataset/ModalitiesMetaDataBlock.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import Markdown from 'markdown-to-jsx'
+
+export interface ModalitiesMetaDataBlockProps {
+  heading: string
+  isMarkdown?: boolean
+  item: React.ReactNode | string[] | number
+  className?: string
+}
+
+export const ModalitiesMetaDataBlock = ({
+  heading,
+  isMarkdown,
+  item,
+  className,
```

Comment:
Since this component is specific to a single feature, i think we can just hardcode these props.